meta-resin-common/classes/kernel-resin.bbclass

```diff
@@ -133,6 +133,7 @@ RESIN_CONFIGS[docker] ?= " \
     CONFIG_EXT4_FS_SECURITY=y \
     CONFIG_AUFS_FS=y \
     CONFIG_AUFS_XATTR=y \
+    CONFIG_KEYS=y \
```

Comment:
Is there any link we can add in the commit log which shows this dependency of docker?